src/Actions/Base/AbstractAction.php

```
@@ -2,83 +2,28 @@
  */
 abstract class AbstractAction
 {
     /**
      * AbstractAction constructor.
      *
-     * @param  Client  $client  The configured Guzzle HTTP client.
-     * @param  ResponseHandler  $responseHandler  The handler responsible for parsing API responses.
+     * @param  HttpTransporter  $transporter  The internal engine used to send
+     *                                        PSR-compliant requests and handle responses.
      */
     public function __construct(
```

PHPStan (2.1.37)

[error] PHPDoc tag `@param` for parameter $transporter with type AsaasPhpSdk\Actions\Base\HttpTransporter is not subtype of native type AsaasPhpSdk\Support\Http\Interface\HttpTransporterInterface.


(parameter.phpDocType)